features/support/env.rb

```diff
@@ -0,0 +1,21 @@
+# Generated by cucumber-sinatra. (2015-03-14 14:51:53 +0000)
+
+ENV['RACK_ENV'] = 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.